addons/hr_skills/static/tests/many2one_avatar_employee.test.js

```diff
@@ -0,0 +1,96 @@
+import {
+    click,
+    contains,
+    openKanbanView,
+    start,
+    startServer,
+} from "@mail/../tests/mail_test_helpers";
+import { beforeEach, describe, expect, test } from "@odoo/hoot";
+import { queryAttribute } from "@odoo/hoot-dom";
+import { defineHrSkillModels } from "@hr_skills/../tests/hr_skills_test_helpers";
+
+describe.current.tags("desktop");
+defineHrSkillModels();
+
+beforeEach(async () => {
+    const pyEnv = await startServer();
+    this.data = {};
```

Comment:
Very ugly `this` reference that would not work if it wasn't for this lambda. Better have a let on top of the file.